bin/handler-slack-multichannel.rb

```diff
@@ -321,7 +341,8 @@ def color
       2 => '#FF0000',
       3 => '#6600CC'
     }
-    color.fetch(check_status.to_i)
+    # When the sensu check status is out of bounds, use color for UNK
+    color.fetch(check_status.to_i, '#6600CC')
```

Comment:
I think given the overlap we should break this out into a library method that can be shared to avoid duplication: https://github.com/sensu-plugins/sensu-plugins-slack/blob/4.0.0/bin/handler-slack.rb#L206-L228

Reply:
See following comment

Reply:
Should be resolved by https://github.com/sensu-plugins/sensu-plugins-slack/pull/82/commits/fac988a26e7ca46362ab7ff934c8336c8ddd84c1, please review